<?xml version="1.0"?>
<launch>
  <param name="uav_system_config_filename" value="$(find aerial_autonomy)/param/matrice_airm_mpc_simulator_config.pbtxt" />
  <param name="log_config_filename" value="$(find aerial_autonomy)/param/log_config.pbtxt" />
  <param name="state_machine_config_filename" value="$(find aerial_autonomy)/param/matrice_airm_mpc_simulator_state_machine_config.pbtxt" />
  <param name="event_file" value="$(find aerial_autonomy)/events/pick_place_events" />
  <arg name="log_level" default="1"/>
  <arg name="log_dir" default="$(find aerial_autonomy)/logs"/>
  <node pkg="aerial_autonomy" type="airm_mpc_node" name="airm_mpc_node" output="screen">
    <remap from="~common/event_manager" to="/rqt_gui/event_trigger" />
    <remap from="~common/system_status" to="/rqt_gui/system_status" />
    <env name="GLOG_log_dir" value="$(arg log_dir)"/>
    <env name="GLOG_v" value="$(arg log_level)"/>
    <env name="GLOG_alsologtostderr" value="1"/>
  </node>
  <!-- TODO Should get these TFs from proto somehow! -->
  <node pkg="tf" type="static_transform_publisher" name="camera_quad" args="0.18 0 -.08 1.5708 0 1.9199 /quad /pointgrey_camera 100"/>
  <node pkg="tf" type="static_transform_publisher" name="arm_quad" args="0.17 0 -.109 3.141 -1.5708 0 /quad /arm 100"/>
  <node pkg="joy" type="joy_node" name="joy" output="screen">
    <remap from="/joy" to="/airm_mpc_node/uav/joy"/>
  </node>
</launch>
